src/components/bulk-email-tool/bulk-email-task-manager/ViewEmailModal.jsx

```diff
@@ -1,12 +1,15 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import { Button, Modal } from '@edx/paragon';
 import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import messages from './messages';
+import { BulkEmailContext } from '../bulk-email-context';
+import { copyToEditor } from '../bulk-email-form/data/actions';
 
 function ViewEmailModal({
-  intl, messageContent, isOpen, setModalOpen, copyTextToEditor,
+  intl, messageContent, isOpen, setModalOpen,
 }) {
+  const [, dispatch] = useContext(BulkEmailContext);
```

Comment:
Is there a missing first argument? Or is this an errant comma? Or is this syntax ([, dispatch]) mean something I might not be familiar with?

Reply:
Its meant to be similar to a tuple in python. It does seem a bit odd in javascript, but this component has no need to access the store directly so I left the first piece of the tuple blank.

Reply:
Thanks for the context!